eng/liveBuilds.targets

```diff
@@ -128,7 +128,7 @@
       <MonoIncludeFiles Include="$(MonoArtifactsPath)\include\**\*.*" />
     </ItemGroup>
 
-    <Error Condition="'@(RuntimeFiles)' == ''" Text="The '$(RuntimeFlavor)' subset must be built before building this project." />
+    <Error Condition="'@(RuntimeFiles)' == '' and !('$(RuntimeFlavor)' == 'CoreCLR' and '$(TargetsAppleMobile)' == 'true')" Text="The '$(RuntimeFlavor)' subset must be built before building this project." />
```

Comment:
this sounds wrong to me, I think I hit the same issue when adding the runtime packs which is why I added the `BuildNativeAOTRuntimePack` that makes the RuntimeFiles pick up NativeAOT assets instead of CoreCLR on L80.

You could set that property too, though I wonder if longer-term we should introduce RuntimeFlavor=NativeAOT

Reply:
Thanks, updated!